src/einsteinpy/utils/exceptions.py

```diff
@@ -10,5 +34,12 @@ def __str__(self):
 
 
 class CoordinateError(BaseError):
+    """
+    Error class for invalid coordinate operations.
+    Raised if Metric objects are defined with coordinate system \
+    not supported by geometry.
+
+    """
+
     def __init__(self, *args, **kwargs):
```

Comment:
Copy over the docstring for `BaseError.__init__()` (after review modifications) to `CoordinateError.__init__()`